src/sst/core/timeConverter.h

```diff
@@ -53,17 +59,29 @@ class TimeConverter
      */
     UnitAlgebra getPeriod() const; // Implemented in timeLord.cc
 
+    /**
+     * TimeConverter* returned by the core should *never* be deleted by
+     * Elements. This was moved to public due to needing to support ObjectMaps.
+     */
+    ~TimeConverter() {}
+    TimeConverter() {} // Only needed to simplify serialization
```

Comment:
I think the comment is not true; now that people will be using TimeConverter directly, they will need a public default constructor.

Reply:
Added a new comment